metabase/metabase
# Fix back to dashboard to select correct tab and preserve filters

Closes https://github.com/metabase/metabase/issues/31786

### Description

We cover two things on navigating back from the question to dashboard in this PR:
- preserve filters, which were applied before you navigated from dashboard to the question
- preserve tab, from which you navigated away from dashboard

### How to verify

- go to dashboard with tabs, add a filter, select some tab, not first one, and click on question to drilldown into it
- click "back" button
- make sure that on dashboard filters are restored and the tab you visited last is selected

### Demo
https://www.loom.com/share/4ba8f5eee3ee463f841fd4a871635bc5
https://www.loom.com/share/8b1ebde1f23e428b915e60e9b3c9c59e
